### PR TITLE
Add mediafile.set_as_logo action + tests.

### PR DESCRIPTION
**Pull request by reiterl**
Allow mimetypes are "image/png", "image/jpeg", "image/gif".

### CODE REVIEW

File: openslides_backend/action/mediafile/set_as_logo.py [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+
+@register_action("mediafile.set_as_logo")
+class MediafileSetAsLogoAction(UpdateAction):
+    """
+    Action to set a mediafile as logo.
+    """
+
+    model = Mediafile()
+    schema = {
```
Comment (jsangmeister) [inline openslides_backend/action/mediafile/set_as_logo.py:18]: Please use the default schema. It should never be needed to explicitly define a schema

**Review by jsangmeister — CHANGES_REQUESTED** [review]
Looks good besides the schema remark

File: openslides_backend/action/mediafile/set_as_logo.py (hunk 1 above)
Comment (reiterl) [reply openslides_backend/action/mediafile/set_as_logo.py]: Done.

**Review by jsangmeister — APPROVED** [review]
You can squash and merge

**Comment by reiterl** [comment]
Rebased + Squashed

**Pull request merged**